packages/fxa-auth-server/test/local/routes/devices-and-sessions.js

```diff
@@ -858,8 +877,22 @@ describe('/account/device/commands', () => {
       last: true,
       index: 4,
       messages: [
-        { index: 3, data: { sender: 'sender1', command: 'three' } },
-        { index: 4, data: { sender: 'sender2', command: 'four' } },
+        {
+          index: 3,
+          data: {
+            sender: '99999999999999999999999999999999',
+            command: 'three',
+            payload: {},
+          },
+        },
+        {
+          index: 4,
+          data: {
+            sender: '88888888888888888888888888888888',
+            command: 'four',
+            payload: {},
+          },
+        },
```

Comment:
Most of these changes are for mock data that didn't fit the response schema of its route.